app/controllers/v1/workflow/steps_controller.rb

```diff
@@ -0,0 +1,8 @@
+class V1::Workflow::StepsController < ApiController
+  def show
+    # TODO: identify current user, check if step id is accessible to user
+    @step = Workflow::Instance::Step.find_by(external_identifier: params[:id])
+
```

Comment:
based on the routes, you need to find `@process` first, then do `@process.steps.find_by!`
This makes sure the step is nested under the process.  I think you have to use `params[:id]` and `params[:process]` or something like that?  You'll have to check `rails routes` to see the exact route regex.